src/repository/test.py

```
@@ -0,0 +1,78 @@
+import uuid
```

Ruff (0.12.2)

`uuid` imported but unused

Remove unused import: `uuid`

(F401)